### ISSUE TITLE
upload failures 

### ISSUE THREAD

**Issue by mikemhenry** [isuruf/openmm-wheels#4]
@isuruf it looks like all of the failure in CI are due to duplicate assets, does that mean we need more unique file names? or does it mean that our matrix needs to be trimmed down? I know on conda-forge these would all be separate packages but on pypi they seem to consolidate. As long as its okay that its kinda a race on who wins, and that it would be way more effort than its worth to trim down the matrix, then I am fine with that but I just want to make sure I understand whats going on. 

I wanted to set the upload action to allow failures if the upload failures are okay, but I am guessing that `.github/workflows/conda-build.yml` is created dynamically when the repo is re-rendered, so it would have to be modified post rendering.

**Comment by isuruf** [comment]
Yes, that's unfortunate. We don't need more unique file names and it would be more effort than it's worth to trim down the matrix.
For N python versions and M cuda versions, we have `N x M` jobs in the matrix, but we have only `N + M` unique wheels.

> I wanted to set the upload action to allow failures if the upload failures are okay, but I am guessing that .github/workflows/conda-build.yml is created dynamically when the repo is re-rendered, so it would have to be modified post rendering.

We do some modifications post rendering in `rerender.sh`.

**Comment by mikemhenry** [comment]
Okay good to know! 

> We do some modifications post rendering in conda-forge.yml.

AH! https://github.com/openmm/openmm-wheels/blob/main/extra.yml brilliant :sunglasses: Okay I can make the changes there :rocket:
